### PR TITLE
feat: adds keywords to work items

### PR DESCRIPTION
**Pull request by CoffeeVector**
feat #398 
I think this would be a good addition to the standard. I think it's not uncommon for people to like putting keywords into their resume.

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -143,6 +143,15 @@
...
               "description": "e.g. Increased profits by 20% from 2011-2012 through viral advertising"
             }
+          },
+          "keywords": {
+            "type": "array",
+            "description": "Specify technologies or skills used, e.g. HTML, CSS, Javascript",
+            "additionalItems": false,
+            "items": {
+              "type": "string",
+              "description": "e.g. python, leadership, public speaking"
```
Comment (peternowee) [inline schema.json:153]: Maybe give only one example on this line, to make clear that each array item contains only one skill, not one long string containing multiple skills. Just for clarity.

File: schema.json [hunk 2]
```diff
@@ -143,6 +143,15 @@
               "type": "string",
               "description": "e.g. Increased profits by 20% from 2011-2012 through viral advertising"
             }
+          },
+          "keywords": {
+            "type": "array",
+            "description": "Specify technologies or skills used, e.g. HTML, CSS, Javascript",
```
Comment (peternowee) [inline schema.json:149]: And maybe just remove the examples from this line, to keep it consistent with the `keywords` in other sections (Skills, Interests, Projects), see https://github.com/jsonresume/resume-schema/blob/v1.0.0/schema.json.

**Comment by pflorek** [comment]
closes https://github.com/jsonresume/resume-schema/issues/420

**Comment by anthonyjdella** [comment]
Isn't this basically what was already proposed in https://github.com/jsonresume/resume-schema/pull/409

**Comment by peternowee** [comment]
> Isn't this basically what was already proposed in jsonresume/resume-schema#409

Yes, you are right, though this one here (jsonresume/resume-schema#401) was first and I think is also a bit better because it names the array `"keywords"`, which is consistent with existing use of `keywords` in other resume sections (Skills, Interests, Projects).

**Comment by jacrys** [comment]
Yes, I missed this in my search for existing PRs, likely because my mental space was in "skills" mode so horse blinders and all that.

**Comment by anthonyjdella** [comment]
> > Isn't this basically what was already proposed in #409
> 
> Yes, you are right, though this one here (#401) was first and I think is also a bit better because it names the array `"keywords"`, which is consistent with existing use of `keywords` in other resume sections (Skills, Interests, Projects).

Ah I didn't realize this one was first, my bad.

**Comment by nightscape** [comment]
Anything preventing this from moving forward?

**Comment by CoffeeVector** [comment]
Oh my bad. I've sinced haven't needed to update my resume in a while, so I didn't notice any activity on this. I've made the changes @peternowee suggested if you still want this change.

**Review by pflorek — APPROVED** [review]

**Comment by nightscape** [comment]
I think someone would need to trigger the workflow, right?
@pflorek @peternowee

**Review by pflorek — APPROVED** [review]
🚀

**Comment by pflorek** [comment]
> I think someone would need to trigger the workflow, right? @pflorek @peternowee

I'm not a maintainer, who is in charge?

**Comment by pflorek** [comment]
@evanplaice @thomasdavis

**Pull request closed without merging**